app/views/shared/_plausible.html.erb

```diff
@@ -1,7 +1 @@
-<script defer data-domain="orcid-staging.princeton.edu" src="https://plausible.io/js/script.js"></script>
-<%
-=begin%>
-  window.plausible needs to be defined in order to send plausible events
-<%
-=end%>
-<script>window.plausible = window.plausible || function() { (window.plausible.q = window.plausible.q || []).push(arguments) }</script>
```

Comment:
Removing comment and plausible script after confirming that issues relating to Plausible Events were caused by adblocker.